internal/cri/server/container_checkpoint_linux.go

```diff
@@ -73,8 +71,17 @@ func (c *criService) CheckpointContainer(ctx context.Context, r *runtime.Checkpo
 		// This is the wrong error message and needs to be adapted once
 		// Kubernetes (the e2e_node/checkpoint) test has been changed to
 		// handle too old or missing CRIU error messages.
-		log.G(ctx).WithError(err).Errorf("Failed to checkpoint container %q", r.GetContainerId())
-		return nil, status.Errorf(codes.Unimplemented, "method CheckpointContainer not implemented")
+		errorMessage := fmt.Sprintf(
+			"CRIU binary not found or too old (<%d). Failed to checkpoint container %q",
```

Comment:
suggestion:

```
errMsg := fmt.Sprintf("failed to checkpoint container %q", r.GetContainerId())
```

since the detail message is covered by `err`.

Reply:
https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/checkpoint_container.go#L247